### PR TITLE
Add Command options' meaning

### PR DESCRIPTION
**Pull request by ajithpv**
It is always better to explicitly mention the meaning of each option(s) provided with mender artifact command.
Added the meaning of provided options for update module example.
Changelog: None

Signed-off-by: Ajith P V <ajithpv@outlook.com>

### CODE REVIEW

File: 03.Devices/10.Update-Modules/docs.md [hunk 1]
```diff
@@ -123,6 +123,11 @@ Then create a new Mender Artifact with this as the only payload, for our new `we
 ```bash
 ./mender-artifact write module-image -t $DEVICE_TYPE  -o ~/Downloads/web-file-1.mender -T web-file -n web-file-1.0 -f hello-world
 ```
+* `-t` - The compatible device type of this Mender Artifact.
+* `-o` - The path where to place the output Mender Artifact. This should always have a .mender suffix.
+* `-T` - The update module name. The name should be same of the update module script which is present inside `/usr/share/mender/modules/v3` path.
+* `-n` - The name of the Mender Artifact.
+* `-f` - The path to the file(s) to be sent to the device in the update.
```
Comment (lluiscampos) [inline 03.Devices/10.Update-Modules/docs.md]: Maybe add a note at the end similar to "For more details, see `mender-artifact write module-image --help`"

File: 03.Devices/10.Update-Modules/docs.md [hunk 2]
```diff
@@ -123,6 +123,11 @@ Then create a new Mender Artifact with this as the only payload, for our new `we
 ```bash
 ./mender-artifact write module-image -t $DEVICE_TYPE  -o ~/Downloads/web-file-1.mender -T web-file -n web-file-1.0 -f hello-world
 ```
+* `-t` - The compatible device type of this Mender Artifact.
```
Comment (lluiscampos) [inline 03.Devices/10.Update-Modules/docs.md]: Before the list of options, add something like "The command line options are detailed below:"

File: 03.Devices/10.Update-Modules/docs.md (hunk 1 above)
Comment (ajithpv) [reply 03.Devices/10.Update-Modules/docs.md]: Agreed and added :+1:

**Comment by lluiscampos** [comment]
Great.

Please squash your commits into a single change set, and it will be ready for merging.

**Comment by lluiscampos** [comment]
Great. Merged and cherry-picked to 2.0.x

**Pull request merged**